AdminWebsite/AdminWebsite/ClientApp/src/app/get-audio-file/get-audio-file-vh/hearing-search-results/hearing-search-results.component.html

```diff
@@ -18,7 +18,7 @@ <h2 class="govuk-heading-m">Results</h2>
   </div>
   <div class="govuk-grid-column-one-quarter vh-pl0"></div>
   <hr class="margin-bottom-22 clear-both" />
-  <div *ngFor="let result of results" class="govuk-grid-row">
+  <div *ngFor="let result of results; trackby: result" class="govuk-grid-row">
```

Comment:
Good job adding a trackby here but it requires a function.

https://netbasal.com/angular-2-improve-performance-with-trackby-cc147b5104e5
https://jinalshah999.medium.com/trackby-with-ngfor-directives-in-angular-application-bd4d0db288eb

Reply:
Is there only ever 1 recording for each hearing? If soomething like this this would be good. Not sure about recording as the name
`
 trackByEmpCode(index: number, recording: HearingAudioSearchModel): string {
    return recording.hearingId;
}
`